src/App.js

```diff
@@ -2,15 +2,43 @@ import React from 'react';
 import './App.scss';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => (
+      { count: state.count + 1 }
+    ));
+  };
+
+  add100 = () => {
+    this.setState(state => (
+      { count: state.count + 100 }
+    ));
+  };
```

Comment:
не записывай объект таким образом
если другой программист захочет добавить еще несколько полей, ему будет неудобно
```suggestion
  addOne = () => {
    this.setState(state => ({
      count: state.count + 1
    }));
  };

  add100 = () => {
    this.setState(state => ({
      count: state.count + 100
    }));
  };
```